tests_integration/snowflake_connector.py

```diff
@@ -10,19 +11,41 @@
 
 
 @pytest.fixture(scope="session")
-def snowflake_session():
+def create_database():
+    database_name = f"db_{uuid.uuid4().hex}"
+
+    config = {
+        "application": "INTEGRATION_TEST",
+        "account": _get_from_env("ACCOUNT", True),
+        "user": _get_from_env("USER", True),
+        "password": _get_from_env("PASSWORD", True),
+        "host": _get_from_env("HOST", True),
+        "role": _get_from_env("ROLE", True),
+    }
+    config = {k: v for k, v in config.items() if v is not None}
+    connection = connector.connect(**config)
+
+    connection.execute_string(f"create database {database_name}")
```

Comment:
Why not `execute`?

Reply:
SnowflakeConnection don't have `execute` method